packages/jaeger-ui/src/components/App/TopNav.js

```diff
@@ -93,9 +94,10 @@ export function TopNavImpl(props: Props) {
             );
           }
           const item = ((m: any): ConfigMenuItem);
+          const target = item.openInSameTab ? "_self" : "_blank";
           return (
             <Menu.Item key={item.label}>
-              <a href={item.url} target="_blank" rel="noopener noreferrer">
+              <a href={item.url} target={target} rel="noopener noreferrer">
                 {item.label}
               </a>
             </Menu.Item>
```

Comment:
can this be DRY-ed with the above code into a helper function?

Reply:
Ok, makes sense.

Reply:
Done